### DISCUSSION TITLE
All devices deselected or failed compatiblity check. Closing lolMiner

### DISCUSSION THREAD

**Discussion by hkunz** [Lolliedieb/lolMiner-releases#278] [Q&A]
Hi, please can anyone help me resolve the error "All devices deselected or failed compatiblity check. Closing lolMiner". it's not giving me much info for me to resolve it. i attached a dxdiag file for more info on my GPU and other specs. Here's what i put in the mine_eth_4G.bat file:

@echo off

setlocal enableDelayedExpansion

Rem #################################
Rem ## Begin of user-editable part ##
Rem #################################

set "POOL=asia1.ethermine.org:4444"
set "WALLET=0x585c8401b995Cc2d353707F880a30a0445172659.lolMinerWorker"										

Rem #################################
Rem ##  End of user-editable part  ##
Rem #################################

lolMiner.exe --algo ETHASH --pool !POOL! --user !WALLET! --4g-alloc-size 4024 --keepfree 8 
timeout 10

[DxDiag.txt](https://github.com/Lolliedieb/lolMiner-releases/files/5992926/DxDiag.txt)

**Comment by Rhacknar** [comment]
first off sharing your wallet id probably not a good idea lol. and the --keepfree8 at the end im not sure of that code but either its keeping miner number 8 free or up to 8 miners free try removing that and try again see if that works
